File: C/PRAK304-2210817310007-HafizhPratamaBudiman.c
#include <stdio.h>

int main()
{
    int n;

printf("Inputkan Bilangan cacah dari 1-99 : ");
scanf("%d", &n);

if (n>99) {
    printf("\nAnda Menginput Melebihi Limit Bilangan");
}

else if (n>=20 && n<=99) {
    printf("\nNilai %d yang anda inputkan merupakan bilangan puluhan",n);
}

else if (n>=10 && n<=19) {
    printf("\nNilai %d yang anda inputkan merupakan bilangan belasan",n);
}

else if (n>=1 && n<=9) {
    printf("\nNilai %d yang anda inputkan merupakan bilangan satuan",n);
}

else {
    printf("\nNilai %d yang anda inputkan merupakan bilangan nol");
}
}

File: Python/PRAK304-2210817310007-HafizhPratamaBudiman.py
n=int(input('Inputkan Bilangan cacah dari 1-99 : '))

if (n>99) :
    print('Anda Menginput Melebihi Limit Bilangan')

elif ((n>=20) and (n<=99)) :
    print('Nilai',n,'yang anda inputkan merupakan bilangan puluhan')

elif ((n>=10) and (n<=19)) :
    print('Nilai',n,'yang anda inputkan merupakan bilangan belasan')

elif ((n>=1) and (n<=9)) :
    print('Nilai',n,'yang anda inputkan merupakan bilangan satuan')

else :
    print('Nilai',n,'yang anda inputkan merupakan bilangan nol')